elasticsearch/macos/opendistro-tar-install.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
License year. change in all files from this PR

Reply:
made the change.